src/app/input-validator/input-validator.component.html

```diff
@@ -1,11 +1,11 @@
 @if (property?.invalid && property?.touched) {
```

Comment:
I would just add an ngIf in the wrapper container to check if property is not null, if its not null then it will execute the code also instead of || use ?? as it is used nowadays instead of || which was more frequently used years ago